overlord/devicestate/firstboot.go

```diff
@@ -111,10 +111,9 @@ func populateStateFromSeedImpl(st *state.State) ([]*state.TaskSet, error) {
 		return nil, err
 	}
 
-	var required map[string]bool
+	required := map[string]bool{}
```

Comment:
seems still worth making it at len(reqSnaps) big or len(reqSnaps)+4  (snapd, base, gadget, kernel)